worlds/stardew_valley/strings/region_names.py

```diff
@@ -1,4 +1,4 @@
-class SVRegion:
+class Region:
```

Comment:
Why not keep the SVRegion name? Or maybe even something more distinct. BaseClasses.Region is already a thing, that this is not a subclass of, so confusion may occur.

Reply:
I realized when making the other string constant classes that having the "SV" prefix on all of them would be incredibly obnoxious and pretty much pointless. I figured that the namespace being imported was enough to know what a given "Region" keyword is, especially since the only places it'll be used are in our own apworld (I hope) and that's the only one we'll care about